src/static/css/cf-enhancements.less

```diff
@@ -493,14 +489,14 @@ tbody {
     border-left-width: @grid_gutter-width;
 
     &:before {
-        content: "";
+        display: block;
+        width: 1px;
+        margin-left: -@grid_gutter-width;
         position: absolute;
         top: 0;
         bottom: 0;
-        width: 1px;
-        display: block;
         background-color: @gray-50;
-        margin-left: -@grid_gutter-width;
+        content: "";
```

Comment:
Trivial: I believe this should be `content: '';` here and throughout per https://github.com/cfpb/cfgov-refresh/pull/946/files#diff-4362f4d56b5ae00d0724af49023e680eR1525.